Tests/StencilTests/EnvironmentSpec.swift

```diff
@@ -1,342 +1,392 @@
-          try expect(reporter.renderError(error), file: file, line: line, function: function) == reporter.renderError(expectedError)
-        }
-
-        $0.it("reports syntax error in base template") {
-          childTemplate = try environment.loadTemplate(name: "invalid-child-super.html")
-          baseTemplate = try environment.loadTemplate(name: "invalid-base.html")
-
-          try expectError(reason: "Unknown filter 'unknown'. Found similar filters: 'uppercase'.",
-                          childToken: "extends \"invalid-base.html\"",
-                          baseToken: "target|unknown")
-        }
-
-        $0.it("reports runtime error in base template") {
-          let filterExtension = Extension()
-          filterExtension.registerFilter("unknown", filter: {  (_: Any?) in
-            throw TemplateSyntaxError("filter error")
-          })
-          environment.extensions += [filterExtension]
-
-          childTemplate = try environment.loadTemplate(name: "invalid-child-super.html")
-          baseTemplate = try environment.loadTemplate(name: "invalid-base.html")
-
-          try expectError(reason: "filter error",
-                          childToken: "block.super",
-                          baseToken: "target|unknown")
-        }
-
-        $0.it("reports syntax error in child template") {
-          childTemplate = Template(templateString: """
-          {% extends "base.html" %}
-          {% block body %}Child {{ target|unknown }}{% endblock %}
-          """, environment: environment, name: nil)
-
-          try expectError(reason: "Unknown filter 'unknown'. Found similar filters: 'uppercase'.",
-                          childToken: "target|unknown",
-                          baseToken: nil)
-        }
-
-        $0.it("reports runtime error in child template") {
-          let filterExtension = Extension()
-          filterExtension.registerFilter("unknown", filter: {  (_: Any?) in
-            throw TemplateSyntaxError("filter error")
-          })
-          environment.extensions += [filterExtension]
-
-          childTemplate = Template(templateString: """
-          {% extends "base.html" %}
-          {% block body %}Child {{ target|unknown }}{% endblock %}
-          """, environment: environment, name: nil)
-
-          try expectError(reason: "filter error",
-                          childToken: "target|unknown",
-                          baseToken: nil)
-        }
+final class EnvironmentIncludeTemplateTests: XCTestCase {
+  var environment = Environment(loader: ExampleLoader())
+  var template: Template = ""
+  var includedTemplate: Template = ""
 
-      }
+  override func setUp() {
+    super.setUp()
+
+    let path = Path(#file) + ".." + "fixtures"
+    let loader = FileSystemLoader(paths: [path])
+    environment = Environment(loader: loader)
+    template = ""
+    includedTemplate = ""
+  }
+
+  func testSyntaxError() throws {
+    template = Template(templateString: """
```

Comment:
I don't see profit from using multi-line strings in this and next function 🤷‍♂️

Reply:
It's to avoid having to escape `"` (quotes).

Reply:
I think escaping is fine... Don't we allow using single quotes though?

Reply:
Sure, we could use `'`.

But you'll see in some other tests I've placed the template in multiline strings just to keep the formatting consistent between multiple tests. It's a nice visual queue of "hey, that's a template", or "hey, that's an expected result".